test/e2e/webdriver/README.md

```diff
@@ -0,0 +1,1396 @@
+> </details>
+
+<br>Our framework supports XPath representation in the patterns mentioned below
+
+> <details><summary><b>Basic XPath</b></summary>
+> XPath expressions select nodes or lists of nodes based on attributes such as `id`,`name`,`classname` and these can be directly specified as shown in the syntax.<br>
+>
+> ![Screenshot displays ‘Confirm’ button in the send transaction page](image/xpath.png)
+>
+> Syntax for locating the button element ‘Confirm’ using xpath
+>
+> ```tsx
+> await driver.findClickableElement({ xpath: '//button[@value=‘Confirm’]' });
+> ```
+>
+> </details>
+>
+> <details><summary><b>Contains</b></summary>
+> Locate element with search text using the method `contains` in the XPath expression (mainly used when there is dynamic text)<br>
+>
+> ![Screenshot displays ‘Reject’ button in the send transaction page](image/xpath-contains.png)
+>
+> Syntax for locating the button element that contains text ‘Reject’
+>
+> ```tsx
+> await driver.findClickableElement({ text: 'Reject', tag: 'button' });
+> ```
+>
+> Another syntax for locating the div menu element that contains text ‘Settings’
+>
+> ```tsx
+> await driver.clickElement({ text: 'Settings', tag: 'div' });
+> ```
+>
+> <br>
+> </details>
+>
+> <details><summary><b>With CSS selector</b></summary>
+> Locate element with a combination of the xpath format and using the CSS selector.<br>
+>
+> ![Screenshot displays the Send transaction page](image/xpath-css-selector.png)
+>
+> Syntax for locating the button that contains text ‘Next’ and css attribute selector
+>
+> ```tsx
+> await driver.clickElement({
+>   text: 'Next',
+>   tag: 'button',
+>   css: '[data-testid="page-container-footer-next"]',
+> });
+> ```
+
+<br>
+</details>
+</details>
+
+<details><summary><b>**** Note - locators ****</b></summary>
+
+> Our team utilizes a custom locator identification syntax consisting of Element Type, Identifier Type, and Identifier Value for efficient finding. Adherence to this syntax is crucial for maintaining consistency and streamlining our workflow.
+>
+> Selenium syntax for locator declaration
+>
+> ```jsx
+> const passwordBox = await findElement(driver, By.css('#password'));
+> await passwordBox.sendKeys('password123');
+> ```
+>
+> Our framework syntax
+>
+> ```tsx
+> await driver.fill('#password', 'password123');
+> ```
+
+</details>
+
+## Elements
+
+Finding web elements is a fundamental task in web automation and testing, allowing scripts to interact with various components of a web page, such as input fields, buttons, links, and more. One of the element identification functions listed below combines with the use [locators](#locators) to uniquely identify an element on the page.
+
+<details><summary><b>findElement</b></summary>
+<br>
+
+> **`findElement`** function is called on the driver instance, it returns a reference to the first element in the DOM that matches with the provided locator. This value can be stored and used for future element actions.
```

Comment:
```suggestion
> **`findElement`** returns a reference to the first element in the DOM thatthe provided locator matches. 
```

TBH I am not sure what `This value can be stored and used for future element actions.` is supposed to mean - if it's worth keeping, maybe elaborate with an example?